editoast/src/core/infra_state.rs

```diff
@@ -13,11 +15,10 @@ pub struct InfraStateRequest {
     pub infra: Option<i64>,
 }
 
-#[derive(Debug, Clone, Derivative, Deserialize, Serialize)]
-#[derivative(Default)]
+#[derive(Debug, Clone, Copy, Default, Deserialize, Serialize)]
```

Comment:
I think that the `Copy` trait is unused, I tried deleting it and running tests, it did not fail